DataConnectors/Templates/Doc_Template_CEF_Connector.md

```diff
@@ -18,4 +18,4 @@ Configure <<Partner Appliance Name>> to forward Syslog messages in CEF format to
 In this document, you learned how to connect <<Partner Appliance Name>> to Azure Sentinel. To learn more about Azure Sentinel, see the following articles:
 - Learn how to [get visibility into your data, and potential threats](quickstart-get-visibility.md).

```

Comment:
Can you please revert the changes to this file? Its not part of the SonicWall Firewall connector.